content/concepts/payment-types/partial-payments.md

```diff
@@ -8,9 +8,11 @@ labels:
 ---
 # Partial Payments
 
-In the default case, the `Amount` field of a [Payment transaction][] in the XRP Ledger specifies the exact amount to deliver, after charging for exchange rates and [transfer fees](transfer-fees.html). The "Partial Payment" flag ([`tfPartialPayment`](payment.html#payment-flags)) allows a payment to succeed by reducing the amount received instead of increasing the amount sent. Partial payments are useful for [returning payments](become-an-xrp-ledger-gateway.html#bouncing-payments) without incurring additional costs to oneself.
+The sender of any Payment can enable the "Partial Payment" flag and send a payment which delivers less than the `Amount` field indicates. When processing any Payment, use the `delivered_amount` metadata field, not the `Amount` field. The `delivered_amount` is the amount a payment actually delivered.
```

Comment:
I think a couple of links would be useful in this intro.
```
The sender of any [Payment transaction][] can enable the ["Partial Payment" flag](payment.html#payment-flags)
and send a payment which delivers less than the `Amount` field indicates. When processing any Payment, use the
`delivered_amount` metadata field, not the `Amount` field. The `delivered_amount` is the amount a payment
actually delivered.
```

They are redundant, but I think valuable to have right up front. Besides, links are cheap and don't really interfere with readability.

Reply:
added